pkg/controller/istio/istio_controller.go

```diff
@@ -305,24 +305,25 @@ func (r *ReconcileConfig) reconcile(logger logr.Logger, config *istiov1beta1.Ist
 	return reconcile.Result{}, nil
 }
 
-func (r *ReconcileConfig) getIngressGatewayAddress(istio *istiov1beta1.Istio, logger logr.Logger) ([]string, error) {
-	var service corev1.Service
+func (r *ReconcileConfig) setIngressGatewayAddress(istio *istiov1beta1.Istio, logger logr.Logger) ([]string, error) {
```

Comment:
Not sure I understand why this became `setIngressGatewayAddress()` from `getIngressGatewayAddress()`. Could you please elaborate?